storage/filesystem/internal/dotgit/dotgit_setref.go

```diff
@@ -0,0 +1,43 @@
+// +build !siva
```

Comment:
should use a more generic key, like can_truncate or something like this?

Reply:
Yes. I'm going to another pull request more generic.